backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -5416,4 +5416,213 @@ databaseChangeLog:
               remarks: The internal database identifier for this entity.
               constraints:
                 primaryKey: true
-                nullable: false
\ No newline at end of file
+                nullable: false
+  - changeSet:
+      id: add-audit-cols-to-facility-table
+      author: tnd8@cdc.gov
+      changes:
+        - dropTable:
+            tableName: api_user_facility
+        - createTable:
+            tableName: api_user_facility
+            remarks: creates a new table to store the relationship between api_user and facility
+            columns:
+              - column:
```

Comment:
question for my own learning -- could we not have added these columns to the existing table or is it less complicated to just drop it and create the table again?  🤔

Reply:
I tried to do that originally but liquibase was failing because you can't add not-nullable columns to an existing table. looks like I could also try adding a default value instead if folks think that's a better option

Reply:
OOO got it! Thank you for letting me know! I think this is fine with me! was genuinely just curious 😸